lua/luasnip/util/environ.lua

```diff
@@ -30,10 +57,28 @@ function Environ:new(pos, o)
 	return o
 end
 
+function Environ.extend_env(name, val)
+	env_namespaces[name] = val
```

Comment:
What do you think about making the `name may not contain "_"`-condition explicit here? (also, empty name should not cause any issues, right? otherwise that could also be checked)

Reply:
For  `Environ` it could be empty, but seems like the parser does not like variables starting with `_` so I will add a check for both.

Reply:
Ah, right, the parser :grimacing:
(You're talking about the current one, right? neovims might actually not have a problem with that (except completely emtpy variable names, probably))

Reply:
We can relax it after moving to the new parser